app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str)\
+            -> None:
```

Comment:
Consider using more descriptive variable names and splitting the long math operation onto additional variables instead of using a line continuation character '\'. This will improve code readability and adhere to the code style guide.